lintercat/lintercat.com
# Feature/projects content

### What does this PR do?
Add projects captures and copy.

### Where should the reviewer start?
`/src/data/projects.js`

### Does this add new dependencies? (Y/n)
n

### How should this be manually tested?
1. Open the project
2. Make sure all project are there and look cool

### Any background context you want to provide?
n

### Screenshots / Gifs
![jan-13-2019 19-24-36](https://user-images.githubusercontent.com/5368213/51093430-fc649a80-1768-11e9-97cc-ebbb1756db12.gif)
